docker-compose.yml

```diff
@@ -24,17 +46,23 @@ services:
       - zookeeper
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
+    networks:
+      - app-network
+
   mongodb:
     image: mongo:latest
-    container_name: mongodb-container
+    container_name: mongodb
```

Comment:
**INSIDE Listener:**
**Purpose**: This is used for communication between Kafka brokers and clients (like producers and consumers) that are within the Docker network. It is typically configured to listen on a specific internal IP address or hostname.
**Example Usage:** If you have other services running in the same Docker network (e.g., your Spring Boot application), they will connect to Kafka using the INSIDE listener.
**OUTSIDE Listener:**
**Purpose**: This listener is exposed to the outside world, allowing clients outside of the Docker network (such as applications running on your local machine) to connect to Kafka. This typically listens on localhost or a specific external IP address.
**Example Usage:** If you are testing Kafka from your local machine or another service outside the Docker network, you will connect using the OUTSIDE listener.